src/sql/src/normalize.rs

```diff
@@ -107,23 +109,60 @@ pub fn op(op: &Op) -> Result<&str, PlanError> {
     Ok(&op.op)
 }
 
+#[derive(Debug)]
+pub enum SqlMaybeValueId {
```

Comment:
There are a bunch of places you're pulling out things from the inner `Value`, e.g. strings--seems like a `TryInto` would be nice